tools/vast-regenerate/main.cpp

```diff
@@ -0,0 +1,306 @@
+//    _   _____   __________
```

Comment:
I cannot comment on file that were only renamed, but the `vast-regenerate` README still calls the tool `mdx-regenerate`.